src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/MvcForm.cs

```diff
@@ -8,11 +8,18 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Represents an HTML form element in an MVC view.
```

Comment:
I'm mostly concerned with us being consistent, which we aren't, at all. The "Represents..." wording has traditionally been used for .NET API docs.